dcos-log/api/v1/handlers.go

```diff
@@ -17,6 +17,15 @@ import (
 	"github.com/gorilla/mux"
 )
 
+type key int
+
+var streamKey key = 1
+
+func requestStreamKeyFromContext(ctx context.Context) bool {
```

Comment:
This seems to be new. Why wasn't it needed in the old implementation. What's changed now to require it?